### PR TITLE
Make 'oc adm prune auth' can prune ServiceAccount

### PR DESCRIPTION
**Pull request by barleyer**
As title,since this **[BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1724596)**,I think this PR can make some sense.

This can prune scc,clusterrolebinding and rolebinding of sa; you can use  `oc adm prune auth sa " " -n " "` to prune specific sa.

### CODE REVIEW

**Comment by barleyer** [comment]
@deads2k @enj could you please help to take a look?

File: pkg/cli/admin/prune/auth/serviceaccount.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		if len(retainedUsers) != len(scc.Users) {
+			updatedSCC := scc
+			updatedSCC.Users = retainedUsers
+			if _, err := securityClient.Update(&updatedSCC); err != nil && !kerrors.IsNotFound(err) {
+				errors = append(errors, err)
+			} else {
+				fmt.Fprintf(out, "securitycontextconstraints.security.openshift.io/"+updatedSCC.Name+" updated\n")
+			}
+		}
+	}
```
Comment (soltysh) [inline pkg/cli/admin/prune/auth/serviceaccount.go]: What about groups?

Comment (soltysh) [reply pkg/cli/admin/prune/auth/serviceaccount.go]: I mean any group you can add scc to through `oc adm policy add-scc-to-group` invocation.

Comment (barleyer) [reply pkg/cli/admin/prune/auth/serviceaccount.go]: We already that feature,see below:
```
# oc adm prune auth -h
Removes references to the specified roles, clusterroles, users, and groups.
Other types are ignored
```

Comment (barleyer) [reply pkg/cli/admin/prune/auth/serviceaccount.go]: And I just add serviceaccout to the list.

Comment (soltysh) [reply pkg/cli/admin/prune/auth/serviceaccount.go]: I mean you're missing the code for removing your particular SCC from groups, compare your code with https://github.com/openshift/oc/blob/master/pkg/cli/admin/prune/auth/user.go#L55

**Pull request closed without merging**